package/alpine-full/Dockerfile

```diff
@@ -3,11 +3,20 @@
 ARG http_proxy
 ARG https_proxy
 
-FROM runiac/deploy:latest-alpine-azure
+FROM runiac/deploy:latest-alpine-aws
```

Comment:
We actually liked the base image being the largest to speed up build time - would there a runtime impact of that I'm not aware of?